.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: npm install
+
```

Comment:
Prob add npm test (runs Jest tests)

Reply:
Let's add that in your PR.

Reply:
Im having issues running nmp test on your branch.

Reply:
Any error?

Reply:
Not really just need to update the README.md so others can also run this locally.